File-Formats/XlsIO/Working-with-Template-Markers.md

```diff
@@ -257,7 +257,7 @@ using (ExcelEngine excelEngine = new ExcelEngine())
 
 The following screenshot represents generated Excel file in which the array of data is bounded.
 
-<img src="Working-with-Template-Markers_images/Bind_data_from_Array_to_Excel_Spreadsheet_Template.jpeg" alt="Bind from Array Example" width="100%" Height="Auto"/>
+<img src="working-with-template-markers_images/bind-data-from-array-to-excel-spreadsheet-template.jpeg" alt="Working with Template Markers in File Formats XLSIO Bind from Array File" width="100%" Height="Auto"/>
```

Comment:
For Image alt text, Don't use "Working with Template Markers in File Formats XLSIO".
Just describe the image functionality shortly.

Example of data bind from array using XLSIO template marker.

Check and correct the all alt text.

Reply:
All alt texts corrected